app/models/respondent.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+
+# A respondent is the person who the claimant is claiming against
+# A claim can have many respondents
+class Respondent < ApplicationRecord
+  belongs_to :address
```

Comment:
It will be added as required - the trouble is an address can 'belong to' many things - claimant, respondent etc... 
Until there is a need to navigate from an address to a respondent - then I can't see the point in adding it ?

Reply:
In that case, a [Polymorphic association](http://guides.rubyonrails.org/association_basics.html#polymorphic-associations) would fit perfectly. Thoughts?

Reply:
Polymorphics are bad for eager loading so use with caution - so only if the need arises for this link going backwards should it get added otherwise we are adding stuff when we don't need it yet which doesn't feel good :-(